src/app/core/top-panel/top-panel.component.ts

```diff
@@ -53,15 +53,17 @@ export class TopPanelComponent implements OnInit {
   }
 
   public goToWorkspaceOrganization(): void {
-    this.store.dispatch(new OrganizationsAction.Select({organizationCode: this.workspace.organizationCode}));
-    this.store.dispatch(new ProjectsAction.Select({projectCode: null}));
+    // TODO
+    // this.store.dispatch(new OrganizationsAction.Select({organizationCode: this.workspace.organizationCode}));
+    // this.store.dispatch(new ProjectsAction.Select({projectCode: null}));
```

Comment:
How do you suggest to handle this?

Reply:
What about storing only `id`s and throw away codes and implement guard to fetch selected organization and selected project on some root component, and when some doesn't exist, redirect to `workspace`. I think we can still use codes in url, but with this way we know, that these codes really exists in DB. What do you think?